vernac/vernacextend.ml

```diff
@@ -72,7 +71,7 @@ type plugin_args = Genarg.raw_generic_argument list
 (* Table of vernac entries *)
 let vernac_tab =
   (Hashtbl.create 211 :
-    (Vernacexpr.extend_name, bool * (plugin_args -> vernac_command)) Hashtbl.t)
+    (Vernacexpr.extend_name, bool * (plugin_args -> ast:Vernacexpr.vernac_expr -> vernac_command)) Hashtbl.t)
```

Comment:
Is it possible to include the `ast` argument directly inside `vernac_command`? I think it'd be cleaner.

Reply:
I think I could not because of the GADT thing, but I can try again.